source/release-notes.txt

```diff
@@ -10,6 +10,68 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.36.1
+----------------
+
+*Released March 15, 2023*
+
+New Features:
+
+- Enable focus mode (:issue:`COMPASS-6474`)
+
+`Full Changelog is available on GitHub 
+<https://github.com/mongodb-js/compass/compare/v1.36.0...v1.36.1>`__
+
+
+|compass| 1.36.0
+----------------
+
+*Released March 14, 2023*
+
+New Features:
+
+- Add stage button between stages (:issue:`COMPASS-6382`)
+- Use type from last array element when inserting new element to array 
+  (:issue:`COMPASS-6432`)
+- Redirect to the new collection after creating it (:issue:`COMPASS-6019`)
+- Stage toolbar (:issue:`COMPASS-6381`)
+- LG darkmode support and UI cleanup in the explain tab (:issue:`COMPASS-6463`)
+- Adds a cancellable loader to explain
+- Enable column store indexes for MongoDB 6.3 (:issue:`COMPASS-6487`)
+- flexi bucket options for Timeseries
+- Upgrade mongosh to 1.7.0
+- Include preview rows in the listCSVFields() result (:issue:`COMPASS-6422`)
+- Enable focus mode (:issue:`COMPASS-6474`)
+- When dropping a collection or database, redirect to either the database or 
+  databases view (:issue:`COMPASS-6018`, :issue:`COMPASS-6434`)
+- Dark theme improvements in the settings modal (:issue:`COMPASS-6552`)
+- Conditional confirmation modal (:issue:`COMPASS-6355`)
+- Adds the refresh CTA to sidebar
+- Open file input before import modal (:issue:`COMPASS-6535`)
+- Enable LG darkmode as public preview (:issue:`COMPASS-6515`, 
+  :issue:`COMPASS-6556`)
+- Hook for keyboard shortcuts (:issue:`COMPASS-6551`)
+- Adds refresh CTA on database and collection list view (:issue:`COMPASS-6431`)
+- Place settings under the most idiomatic menu for the platform 
+  (:issue:`COMPASS-6430`)
+
+Bug Fixes:
+
+- If a date is in the safe range, go with relaxed EJSON rather than canonical 
+  (:issue:`COMPASS-5744`)
+- Redesign of add stage button (:issue:`COMPASS-6449`)
+- Optimises the opening of tab
+- Don't show negative count on delete when no document count (:issue:`COMPASS-5996`)
+- Stop on errors when stopOnErrors is true (:issue:`COMPASS-6518`)
+- Undefined rather than false if getCloudInfo fails, support SRV URIs 
+  (:issue:`COMPASS-6111`)
+- Cancel edit on non-existent field (:issue:`COMPASS-6505`)
+- Halt autoupdater on application exit to prevent logger crashing 
+  (:issue:`COMPASS-6051`)
+- Do not reset stage value if it was already changed (:issue:`COMPASS-6584`)
+
+`Full Changelog available on Github: v1.35.0...v1.36.0 
```

Comment:
The other links on the page don't have this part: v1.35.0...v1.36.0

Reply:
Fixed